BestPractices/Source/Detail/View/DetailView.swift

```diff
@@ -23,38 +23,25 @@ class DetailView: UIView {
         return label
     }()
 
-    var interitemSpacingConstraints: [Constraint] = []
+    private(set) lazy var stackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [
+            title,
+            button,
+            selectionResult,
+        ])
+        stackView.axis = .vertical
+        stackView.alignment = .center
+        return stackView
+    }()
 
     let requiresConstraintBasedLayout = true
 
     override init(frame: CGRect) {
         super.init(frame: frame)
 
-        backgroundColor = .white
-
-        addSubview(title)
-        title.snp.makeConstraints { make in
+        addSubview(stackView)
+        stackView.snp.makeConstraints { make in
             make.center.equalTo(self)
```

Comment:
Tiny nit, but wouldn't it be "more" correct to constrain the stackView edges instead of just center still?

Not sure if it matters for this demo project though

Reply:
Yeah, I might have to make some layout rule changes to make it look correct. I'll try it out.

Reply:
Hm, it looks like if the stack view's frame is as tall as the view it will spread the elements out. What layout rules would you use to force all elements to center vertically within the frame when the axis is vertical?

Reply:
Ah, I see. I'm not sure without playing with it. Was just concerned about strings possibly running off the edges if they were long. But is this already handled by the stackView itself?

Reply:
Gonna resolve this since I don't see a way to use a full height stack view that lays out the same as this implementation.